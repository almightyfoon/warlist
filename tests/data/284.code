ca501b_-0tcVaPaPbvbfaKbgaObebeb9br___-0tg-aQbvaKbobgaObeb9bkbl