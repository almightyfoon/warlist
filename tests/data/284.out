https://conflictchamber.com/#ca501b_-0tcVaPaPbvbfaKkNaObebeb9br___-0tg-kHbvaKkMkNaObeb9bkbl

Legion Army - 73 / 75 points
[Theme] Oracles of Annihilation

[Lylyth 3] Lylyth, Reckoning of Everblight [+29]
 - Angelius [17]
 - Angelius [17]
 - Naga Nightlurker [8]
 - Nephilim Bolt Thrower [11]
 - Nephilim Protector [10]
 - Scythean [16]
 - Seraph [14]
Blighted Nyss Sorceress & Hellion [0(6)]
Blighted Nyss Sorceress & Hellion [0(6)]
The Forsaken [0(4)]
Blackfrost Shard [9]

Legion Army - 73 / 75 points
[Theme] Oracles of Annihilation

[Kallus 2] Kallus, Devastation of Everblight [+28]
 - Carnivean [18]
 - Naga Nightlurker [8]
 - Nephilim Protector [10]
 - Ravagore [16]
 - Scythean [16]
 - Seraph [14]
Blighted Nyss Sorceress & Hellion [0(6)]
The Forsaken [4]
Blighted Nyss Hex Hunters (max) [15]
 - Bayal, Hound of Everblight [0(6)]
